src/Views/Page.elm

```diff
@@ -146,13 +147,13 @@ headerMenuLinks =
 footerMenuLinks : List MenuLink
 footerMenuLinks =
     [ Internal "Accueil" Route.Home Home
-    , Internal "Simulateur" (Route.TextileSimulator Impact.defaultTextileTrigram Unit.PerItem ViewMode.Simple Nothing) TextileSimulator
+    , Internal "Simulateur textile" (Route.TextileSimulator Impact.defaultTextileTrigram Unit.PerItem ViewMode.Simple Nothing) TextileSimulator
     , Internal "Exemples" Route.TextileExamples TextileExamples
     , Internal "Explorateur" (Route.Explore Scope.Textile (Dataset.Countries Nothing)) Explore
     , Internal "API" Route.Api Api
     , Internal "Nouveautés" Route.Changelog Changelog
     , Internal "Statistiques" Route.Stats Stats
-    , Internal "Alimentaire" (Route.FoodBuilder Impact.defaultFoodTrigram Nothing) FoodBuilder
+    , Internal "Simulateur alimentaire" (Route.FoodBuilder Impact.defaultFoodTrigram Nothing) FoodBuilder
```

Comment:
We should probably rather have `Textile` and `Alimentaire` links next to each others — so we match the order of the links in the header 

![image](https://user-images.githubusercontent.com/41547/227496588-1973a09f-1256-4dac-8fcc-cb370477a039.png)

So we read `Accueil | Textile | Alimentaire | …`